frontend/src/v4/routes/components/dialogContainer/components/dialog/dialog.component.tsx

```diff
@@ -133,33 +136,22 @@ export const Dialog: FunctionComponent<IProps> = forwardRef((props, ref: Ref<HTM
 		}
 	};
 
-	const renderV5Dialog = () => {
-		const data = { content, ...(props.data || {})};
-
-		return (
-			<AlertModal
-				{...data}
-				handleResolve={handleResolve}
-				handleClose={handleClose}
-				dialogId={props.id}
-				handleDisableClose={handleCloseDisable}
-				disableClosed={closeDisabled}
-			/>
-		);
-	};
+	const renderV4ThemeProviders = ({ children }) => (
```

Comment:
Should this be _renderV5ThemeProviders_?

Reply:
Fixed